examples/webgl_materials_envmaps_hdr.html

```diff
@@ -114,7 +116,9 @@
 				};
 
 				var hdrUrls = genCubeUrls( './textures/cube/pisaHDR/', '.hdr' );
-				new THREE.HDRCubeTextureLoader().load( THREE.UnsignedByteType, hdrUrls, function ( hdrCubeMap ) {
+				new THREE.HDRCubeTextureLoader().load( THREE.UnsignedByteType, hdrUrls, function ( _hdrCubeMap ) {
+
+					hdrCubeMap = _hdrCubeMap;
```

Comment:
Can you please explain the reason for this pattern? Can you restore the former pattern and still get the demo to work?

Reply:
Which pattern specifically? We do need to make the CubeTexture global to set to scene.background later on, and this just involved the least amount of changes -- happy to change it to anything else